.husky/commit-msg

```diff
@@ -2,3 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npx --no -- commitlint --edit 
+npx --no -- commitlint --edit 
```

Comment:
it seems you've added this line accidentally

Reply:
i assume it was added by the install from cli i'll take a look thanks